gitgud/skills/level_builder.py

```diff
@@ -74,8 +75,8 @@ def test_failed(self):
           
 
 class BasicLevel(Level):
-    def __init__(self, name, skill_package):
-        super().__init__(name)
+    def __init__(self, name, skill_package, readable_name=""):
```

Comment:
Don't give this a default value. We want everything to have a human readable name